vars/Fedora.yml

```diff
@@ -20,3 +20,4 @@ packages:
   - xz-devel
   - zchunk-devel
   - zlib-devel
+  - swig
```

Comment:
For all the lists, please put this in alphabetical order (the 1st few packages are not, but you'll see.)